### ISSUE TITLE
Allow editing all config files

### ISSUE THREAD

**Issue by AD7six** [nginx/docker-extension#4]
I took a look at this extension, looks promising :).

At the moment though it is only possible to see/edit config files under `/etc/nginx` - any files that are included from other locations appear as a blue bar in the configuration editor:

<img width="1394" alt="Screenshot 2023-03-22 at 09 29 34" src="https://user-images.githubusercontent.com/33387/226853039-ee24e7fc-8741-4ed0-969d-4faf8108f165.png">

Taking a look at [the code](https://github.com/nginx/docker-extension/blob/32d687a520ef9890d8a9bdc9e5b6fa53a34cd745/ui/src/configurationEditor/ConfigurationEditor.tsx#L51) this is a deliberate restriction:

```
        const getConfiguration = async () => {
            instanceService.getConfigurations(props.nginxInstance.id).then((data: any) => {
                let filesArray: Array<string> = data
                filesArray = filesArray.filter(item => item != "").map((item: string) => {
                    let match = item.match('(?:\\/etc\\/nginx\\/.*)'); #<-----
                    if (match != undefined) {
                        return match[0].replace(`:`, ``)
                    } else {
                        return ""
                    }
                })
                setConfiguration(filesArray)
            })
```

Note that:
* That regex doesn't have a `^` anchor so it actually matches anything with `/etc/nginx/` anywhere in its path. As a test I included `/local/etc/nginx/test.conf` and it appeared in the config file selector as `/etc/nginx/test.conf` (didn't show anything though because _that_ path doesn't exist)
* `return ""` results in a blank line, would be preferable to have no output (using a filter, I guess :) ) if that's the intent

I'm not 100% sure of the reason for that path restriction but in my case it's problematic because our consul-templated config files are source from a different directory:

```
root@local-server:/# nginx -T | grep 'configuration file'
nginx: the configuration file /etc/nginx/nginx.conf syntax is ok
nginx: configuration file /etc/nginx/nginx.conf test is successful
# configuration file /etc/nginx/nginx.conf:
# configuration file /local/nginx/conf.d/blue-one.conf:                         # <------
# configuration file /etc/nginx/mime.types:
# configuration file /etc/nginx/common/logging.conf:
# configuration file /etc/nginx/common/logging-variables.conf:
# configuration file /local/nginx/http.d/blue-two.conf:                        # <------
# configuration file /local/nginx/http.d/blue-three.conf:                      # <------
# configuration file /local/nginx/proxy-configs/blue-four.conf:                # <------
# configuration file /local/nginx/sites-enabled/blue-five.conf:                # <------
# configuration file /etc/nginx/proxy-configs/envoy-proxy-pass.conf:
# configuration file /local/nginx/sites-enabled/blue-six.conf:                 # <------
# configuration file /etc/nginx/common/brotli.conf:
# configuration file /etc/nginx/common/gzip.conf:
# configuration file /etc/nginx/common/geoip.conf:
root@local-server:/#
```

So the ask here is either:
* Remove the 'configuration file' path restriction
* Make the `/etc/nginx/` restriction configurable, and allow multiple root paths

**Comment by tippexs** [comment]
Thanks for reporting this. Will work on it.

**Comment by tippexs** [comment]
solved in 0.0.3
